FF8.Core/Sources/Coords3D.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace FF8.Core
+{
+    public struct Coords3D
+    {
+        public readonly Int32 X;
+        public readonly Int32 Y;
+        public readonly Int32 Z;
+
+        public Coords3D(Int32 x, Int32 y, Int32 z)
+        {
+            X = x;
+            Y = y;
```

Comment:
This may interfere with different coordinate system (X -Z Y). Let's just leave that as-is for testing.